app/controllers/application_controller.rb

```diff
@@ -3,8 +3,6 @@ class ApplicationController < ActionController::API
 
   before_action :authorize!
 
-  rescue_from ActiveRecord::RecordNotFound, with: :handle_not_found
```

Comment:
Rails does this automatically in production and returns a 404 with the following body:

```
{
    "status": 404,
    "error": "Not Found"
}
```